Assets/_BForBoss/_Core/Scripts/Environment/OffMeshLinkChecker.cs

```diff
@@ -0,0 +1,63 @@
+using UnityEngine;
+using UnityEngine.AI;
+
+namespace BForBoss
+{
+    public class OffMeshLinkChecker : MonoBehaviour
+    {
+        [SerializeField] private NavMeshAgent _agent = null;
+        [SerializeField, Tooltip("How much should the cost of a mesh link multiply by when in use?")]
+        private float _multiplier = 1000f;
```

Comment:
Does this need to be dynamic variable? or just some random high value?

Reply:
I guess this would work if you want enemies to come in a certain order.

Reply:
yeah the high value is to discourage the enemies from waiting for an enemy to be done using this offmeshlink instead of trying to find a different path with a lower cost.